translate/google/cloud/translate/client.py

```diff
@@ -173,7 +185,11 @@ def translate(self, values, target_language=None, format_=None,
                                   for translation. Sets the ``cid`` parameter
                                   in the query.
 
-        :rtype: str or list list
+        :type model: str
+        :param model: (Optional) The model used to translate the text. The
+                      only accepted values are ``base`` and ``nmt``.
```

Comment:
Please do, I definitely prefer that over magic strings. Unless there is some way to get the list of models from the API?